hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/AllTopicClientsEndpoint.java

```diff
@@ -0,0 +1,19 @@
+package pl.allegro.tech.hermes.api.endpoints;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+
+import java.util.List;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("topics")
+public interface AllTopicClientsEndpoint {
+
+    @GET
+    @Produces(APPLICATION_JSON)
+    @Path("/{topic}/clients")
+    List<String> getTopicClients(@PathParam("topic") String topic);
```

Comment:
What about adding this method to [TopicEndpoint](https://github.com/allegro/hermes/blob/master/hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/TopicEndpoint.java)? It is already registered on path `@Path("topics")` :)

Reply:
I would like to keep it away from TopicEndpoint, but I have changed the path to be unique, like it is for example in [SchemaEndpoint](https://github.com/allegro/hermes/blob/master/hermes-api/src/main/java/pl/allegro/tech/hermes/api/endpoints/SchemaEndpoint.java)